src/MSALWrapper/AuthMode.cs

```diff
@@ -27,15 +27,20 @@ public enum AuthMode : short
         /// </summary>
         Broker = 1 << 2,
 
+        /// <summary>
+        /// Integrated Windows auth mode.
+        /// </summary>
+        IWA = 1 << 3,
+
         /// <summary>
         /// All auth modes.
         /// </summary>
-        All = Broker | Web | DeviceCode,
+        All = IWA | Broker | Web | DeviceCode,

```

Comment:
I'd be more comfortable is the non-existent mode was not added to this fields used by the CLI already, though it is a not a breaking change given it just won't be created.

Reply:
(This is not a request for changes, I'm okay to approve as is).